packages/amplify-cli-core/src/errors/amplify-exception.ts

```diff
@@ -123,6 +123,7 @@ export type AmplifyFaultType =
   | 'AmplifyBackupFault'
   | 'BackendPullFault'
   | 'BackendDeleteFault'
+  | 'ConfigurationFault'
```

Comment:
possible follow up. Should `Fault` and `Error` be separate dimension? Looks like we're mixing something I'd call area or component with severity.